### DISCUSSION TITLE
Ask Position X and Y

### DISCUSSION THREAD

**Discussion by karomahlailatul** [ValentinH/react-easy-crop#556] [Q&A]
![image](https://github.com/user-attachments/assets/8e5f51cf-871e-4a2b-8887-98ffa98c91ca)

hi, I am lay, 
I want to take a position with restrict position false, but always going into the center, maybe I can know why it is going to the bottom automatically, 

croppedAreaPixels {
    "width": 1702,
    "height": 3026,
    "x": 0,
    "y": -233
}
mediaSize {
    "width": 150,
    "height": 225.6169212690952,
    "naturalWidth": 1702,
    "naturalHeight": 2560
}

**Comment by ValentinH** [comment]
What initial value do you pass to the `crop` prop?

**Reply by karomahlailatul** [reply]
initially like this, default, I did not change anything, can it possible us to just like set x and y into the bottom position based on croppedAreaPixel, like proper with the guideline I created,
``` 
const [crop, setCrop] = useState({ x: 0, y: 0 });
```
thank you

**Reply by ValentinH** [reply]
You can use the `initialCroppedAreaPixels` I think.

**Reply by karomahlailatul** [reply]
initialCroppedAreaPixels had the same default was 0, 0, but it i tried again, it always going into the center, whatever image I change the image, 

![image](https://github.com/user-attachments/assets/ff2de879-e688-4bd8-8d61-eb561477f3c6)
croppedAreaPixels {
    "width": 1500,
    "height": 2667,
    "x": 0,
    "y": -606
}
Media Size {
    "width": 150,
    "height": 145.50000000000003,
    "naturalWidth": 1500,
    "naturalHeight": 1455
}

**Reply by ValentinH** [reply]
What value do you pass to the initial prop?

**Reply by karomahlailatul** [reply]
``` 
const [initialCroppedAreaPixels, setInitialCroppedAreaPixels] = useState({
    x: 0,
    y: 0,
    width: 1080,
    height: 1920,
  });
```
    it based on image upload, so maybe initialCroppedAreaPixels not can change anything,

**Reply by ValentinH** [reply]
You need to set the y position accordingly to have the image at the bottom.

**Reply by karomahlailatul** [reply]
still need to set manual changes right, we cannot choose an automatic position. in center bottom or on top

**Reply by ValentinH** [reply]
No, you have to compute it on your own indeed.

**Reply by karomahlailatul** [reply]
ahh i see, thank you,
